cerebrium/endpoints/custom-domains.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: "Custom Domains"
+description: "Connect your own domain to your Cerebrium project for professional, branded API endpoints"
+---
+
+Custom domains allow you to serve your Cerebrium apps through your own domain instead of the default `*.cerebrium.ai` URLs. Once configured, your API calls will use your custom domain while keeping the same path structure: `api.yourdomain.com/v4/p-1234/my-app/run`.
+
+## Key Features
+
+- Support for apex domains (`yourdomain.com`), subdomains (`api.yourdomain.com`), and wildcard domains (`*.yourdomain.com`)
+- Automatic SSL certificate provisioning and renewal
+- Project-level domains - one domain serves all apps in your project
+- Multiple domains can point to the same project
+- Professional branding with your own domain instead of `*.cerebrium.ai` URLs
+
+## How Custom Domains Work
+
+Custom domains in Cerebrium are **project-level**, meaning:
+
+- **One domain serves all apps** in your project
+- **Multiple domains** can point to the same project (useful for different environments or regions)
+- **Professional branding**: Use your own domain instead of `*.cerebrium.ai` URLs
+- **Region-specific**: Each domain is tied to a specific AWS region for optimal performance
```

Comment:
You mention each domain services an entire project (all apps) but projects can have apps in multiple regions.

Also you mention regions in point 2 so a little confused